### PR TITLE
Interactive Validator Client Onboarding

### PR DESCRIPTION
**Pull request by rauljordan**
Part of #1586 

---

# Description

**Write why you are making the changes in this pull request**

This PR allows users to spin up validator clients by simply calling `bazel run //validator` with no command flags. If an account does not exist, the service will guide the user and create a new account. We securely read the password bytes from stdin using the golang/x/ssh package, specify a default suggested datadir for the validator keystore, and allow user to specify the number of validators to launch.

### CODE REVIEW

File: validator/types/flags.go [hunk 1]
```diff
@@ -22,13 +28,40 @@ var (
 		Usage: "Certificate for secure gRPC. Pass this and the tls-key flag in order to use gRPC securely.",
 	}
 	// KeystorePathFlag defines the location of the keystore directory for a validator's account.
-	KeystorePathFlag = cli.StringFlag{
+	KeystorePathFlag = cmd.DirectoryFlag{
```
Comment (prestonvanloon) [inline validator/types/flags.go]: Nice

**Comment by terencechain** [comment]
Tests failing. But everything else looks good.
Except to try this on docker

**Comment by prestonvanloon** [comment]
Lint failures in travis
```
validator/accounts/account.go:95::warning: Potential file inclusion via variable,MEDIUM,HIGH (gosec)
validator/main.go:84::warning: Errors unhandled.,LOW,HIGH (gosec)
validator/main.go:70:48:warning: parameter result 0 (string) is never used (unparam)
```

File: validator/accounts/account.go [hunk 2]
```diff
@@ -87,3 +89,23 @@ func NewValidatorAccount(directory string, password string) error {
 `, serializedData)
 	return nil
 }
+
+// Exists checks if a validator account at a given keystore path exists.
+func Exists(keystorePath string) (bool, error) {
```
Comment (nisdas) [inline validator/accounts/account.go]: this doesnt check if an account exists, it just checks if there is a directory.

**Pull request merged**